bftengine/src/bftengine/messages/PartialProofsSet.cpp

```diff
@@ -227,18 +227,23 @@ class AsynchProofCreationJob : public util::SimpleThreadPool::Job {
       return;
     }
 
-    acc->getFullSignedData(bufferForSigComputations, sigLength);
+    acc->getFullSignedData(const_cast<char*>(bufferForSigComputations.data()), sigLength);
```

Comment:
This cast won't be needed after removing the const on 218.

Reply:
Fixed.